koku/reporting/provider/gcp/models.py

```diff
@@ -81,3 +101,137 @@ class Meta:
+    usage_to_pricing_units = models.DecimalField(max_digits=24, decimal_places=9, null=True)
+    usage_pricing_unit = models.CharField(max_length=256, null=True, blank=True)
+    invoice_month = models.CharField(max_length=256, null=True, blank=True)
+    tax_type = models.CharField(max_length=256, null=True, blank=True)
+
+
+class GCPCostEntryLineItemDailySummary(models.Model):
+    """A daily aggregation of line items.
+
+    This table is aggregated by service, and does not
+    have a breakdown by resource or tags. The contents of this table
+    should be considered ephemeral. It will be regularly deleted from
+    and repopulated.
+
+    """
+
+    class Meta:
+        """Meta for GCPCostEntryLineItemDailySummary."""
+
+        db_table = "reporting_gcpcostentrylineitem_daily_summary"
+        # TODO: Turn off managed.
+        # It is important to first initially create the table with managed on
+        # Then create another migration to flip them off, that way the table
+        # managed = False
+        indexes = [
+            models.Index(fields=["usage_start"], name="gcp_summary_usage_start_idx"),
+            models.Index(fields=["instance_type"], name="gcp_summary_instance_type_idx"),
+            GinIndex(fields=["tags"], name="gcp_tags_idx"),
+            models.Index(fields=["project"], name="gcp_summary_project_idx"),
+            models.Index(fields=["cost_entry_product"], name="gcp_summary_product_idx"),
+        ]
+
+    uuid = models.UUIDField(primary_key=True)
+
+    cost_entry_bill = models.ForeignKey(GCPCostEntryBill, on_delete=models.CASCADE)
+
+    # The following fields are used for grouping
+    cost_entry_product = models.ForeignKey(GCPCostEntryProductService, null=True, on_delete=models.CASCADE)
+    project = models.ForeignKey(GCPProject, on_delete=models.CASCADE)
+    usage_start = models.DateField(null=False)
+    usage_end = models.DateField(null=True)
+    region = models.CharField(max_length=50, null=True)
+    instance_type = models.CharField(max_length=50, null=True)
+    unit = models.CharField(max_length=63, null=True)
+    line_item_type = models.CharField(max_length=256, null=True)
+    usage_amount = models.DecimalField(max_digits=24, decimal_places=9, null=True)
+    currency = models.CharField(max_length=10)
+
+    # The following fields are aggregates
+    unblended_cost = models.DecimalField(max_digits=24, decimal_places=9, null=True)
+    markup_cost = models.DecimalField(max_digits=24, decimal_places=9, null=True)
+    tags = JSONField(null=True)
+    source_uuid = models.UUIDField(unique=False, null=True)
+
+    # TODO: I am not sure if these are needed or just aws specific. I went ahead
+    # commented them out in case we need them.
+    # unblended_rate = models.DecimalField(max_digits=24, decimal_places=9, null=True)
+    # blended_rate = models.DecimalField(max_digits=24, decimal_places=9, null=True)
+    # blended_cost = models.DecimalField(max_digits=24, decimal_places=9, null=True)
+
```

Comment:
I am still unsure about the comment above ^^^

Reply:
You can safely remove these.